s3repo/repoinfo.py

```diff
@@ -15,3 +15,52 @@ def __hash__(self):
 
     def __eq__(self, other):
         return self.path == self.path
+
+
+class RepoAnatation:
+    """Anatation of the repository.
+
+    This is a more common repository description than "RepoInfo".
+    What does it mean:
+    When I want to add something (or just update the metainformation)
+    of repository "release/1.10/el/7", in fact the three repositories
+    will be updated (x86_64, SRPMS, aarch64), but it's not interesting for
+    me. Or, when I update something in the repository "release/1.10/debian/jessie"
+    the metainformation for all debian distributions will be updated.
+    So, the "RepoAnatation" describes the repository from the user's point of
+    view. And in fact, it can include more than one repository.
```

Comment:
Good description! However I miss one point regarding Debian repositories. It is clear for me what is going on for EL: x86_64, aarch64, SRPMS will be updated. But it is not clear for me what is the list for the Debian repositories. Can you, please, leave an example just like you did for EL?

Reply:
Updated